src/EditorFeatures/VisualBasicTest/LineCommit/CommitWithViewTests.vb

```diff
@@ -821,7 +821,7 @@ Imports System.Linq
 
 Module Program
     Sub Main(args As String())
-        
+
```

Comment:
I'm surprised that there wasn't an IDE0055 warning here.
Is the analyzer disabled for tests?